aptos-node/src/main.rs

```diff
@@ -72,8 +72,17 @@ fn main() {
             rng,
         );
     } else {
-        let config = NodeConfig::load(args.config.unwrap()).expect("Failed to load node config");
+        // Load the config file
+        let config_path = args.config.unwrap();
```

Comment:
If we're improving the output, let's also have a message if no arg was provided (or is that provided by the opt already?)

Reply:
Yeah, that's provided by `opt` already 😄 

```
% cargo run -p aptos-node                             
    Finished dev [unoptimized + debuginfo] target(s) in 0.35s
     Running `target/debug/aptos-node`
error: The following required arguments were not provided:
    --config <config>

USAGE:
    aptos-node [FLAGS] [OPTIONS] --config <config>

```